ui/app/templates/components/secret-create-or-update.hbs

```diff
@@ -160,22 +164,24 @@
   >
     <div class="box is-sideless is-fullwidth is-marginless padding-top">
       <MessageError @model={{@modelForData}} @errorMessage={{this.error}} />
+      <NamespaceReminder @mode="edit" @noun="secret" />
       {{#if (eq @canReadSecretData false)}}
-        <AlertBanner
-          @type="warning"
-          @message="You do not have read permissions. If a secret exists here creating a new secret will overwrite it."
-          data-test-warning-no-read-permissions
-        />
+        <Hds::Alert @type="inline" @color="warning" class="has-bottom-margin-s" data-test-warning-no-read-permissions as |A|>
+          <A.Title>Warning</A.Title>
+          <A.Description>You do not have read permissions. If a secret exists here creating a new secret will overwrite it.</A.Description>
+        </Hds::Alert>
       {{/if}}
-      <NamespaceReminder @mode="edit" @noun="secret" />
```

Comment:
This being in the middle looked a little wild: 
![image](https://github.com/hashicorp/vault/assets/68122737/c6ddb604-2d6b-405f-97eb-e832a7a8b734)